src/v2/hocs/WithIsSpiderRequesting/queries/isSpiderRequesting.ts

```diff
@@ -3,6 +3,7 @@ import gql from 'graphql-tag'
 export default gql`
   query IsSpiderRequestingQuery {
     sharify @client {
+      # FIXME: Where is this constant coming from?
```

Comment:
Noticing this Apollo validation error and wondering if this is a bug?

Reply:
?

Reply:
It's coming from here: https://github.com/aredotna/ervell/blob/master/src/lib/middleware/isSpider.js

Reply:
Fixed by adding a type to [here](https://github.com/aredotna/ervell/blob/master/src/v2/apollo/localState/clientSchema.graphql#L41-L43)